doc/MANUAL.adoc

```diff
@@ -947,10 +947,14 @@ Examples:
 * `http://localhost:8080/`
 * `http://example.com/cache`
 
+Optional attributes:
+
+* *username*: Username for the HTTP Basic Authentication.
+* *password*: Password for the HTTP Basic Authentication.
```

Comment:
In 2de287348651a8f5b416488dd97e33f7dcb8bd80 I chose to extract username and password from the URL instead of using attributes for the Redis backend. Does that sound OK for the HTTP backend as well?

Reply:
I just noticed it when I looked at the timeouts. I'll adjust the PR.